src/main/java/item/ItemManage.java

```diff
@@ -0,0 +1,238 @@
+package item;
+
+import exception.RoboastException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class ItemManage {
+    private final ArrayList<Item> itemList;
+    private static final String LINE = "_".repeat(50);
+    private static final String[] COMMAND_LIST = {"list","mark","unmark","todo","deadline","event","deleteAll"};
+
+    public ItemManage(ArrayList<Item> itemList) {
+        this.itemList = itemList;
+    }
+
+    public void action(String input) throws RoboastException {
+
+        String command;
+        String content;
+        int spaceIndex = input.indexOf(' ');
+
+        try {
+            command = input.substring(0, spaceIndex);
+            content = input.substring(spaceIndex + 1);
+        }
+        catch (IndexOutOfBoundsException e) {
+            command = input;
+            content = "";
+        }
+        if (command.equals("mark")) {
+            mark(content, true);
+        }
+        else if (command.equals("unmark")) {
+            mark(content, false);
+        }
+        else if (command.equals("todo")) {
+            addTodo(content);
+        }
+        else if (command.equals("deadline")) {
+            addDeadlines(content);
+        }
+        else if (command.equals("event")) {
+            addEvents(content);
+        }
+        else if (command.equals("list")) {
+            showItemList();
+        }
+        else if (command.equals("deleteAll")) {
+            deleteAll();
+        }
+        else if (command.equals("delete")) {
+            delete(content);
+        }
+        else {
+            throw new RoboastException("Invalid command");
+        }
+
+    }
+
+    public void mark(String content, boolean isDone) {
```

Comment:
What is exaclt meant by variable content? Is it the input from the user? Consider naming it something else more intuitive.